addons/account/views/account_move_views.xml

```diff
@@ -879,6 +879,56 @@
                                         <span class="o_stat_text">Cash Basis Entries</span>
                                     </div>
                             </button>
+                            <button name="open_adjusting_entries"
+                                    class="oe_stat_button"
+                                    icon="fa-bars"
+                                    type="object"
+                                    invisible="not adjusting_entries_move_ids">
+                                    <field name="adjusting_entries_count" string="Adjusting Entries" widget="statinfo"/>
+                            </button>
+                            <button name="open_adjusting_entry_origin_moves"
+                                    class="oe_stat_button"
+                                    icon="fa-bars"
+                                    type="object"
+                                    invisible="not adjusting_entry_origin_move_ids or adjusting_entry_origin_moves_type != 'out_invoice'">
+                                    <div class="o_stat_info">
+                                        <span class="o_stat_text">Invoice</span>
```

Comment:
Instead of repeating the same block multiple times, you could use <field name="adjusting_entry_origin_label"> directly in here, hiding it if the count is != 1, and instead showing (if the count is > 1) <field name="adjusting_entry_origin_moves_count" string="Invoices" widget="statinfo"/>

Declaration would then be shorter and more robust :)